src/Screens/Gameplay/Components/Actions/ActionsButton.jsx

```diff
@@ -5,10 +5,11 @@ import texts from 'texts';
 import ActionsTab from './ActionsTab/ActionsTab';
 import { useDispatch, useSelector } from 'react-redux';
 import { unmountComponents, mountComponents } from 'Redux/Slices/MountedComponents';
+import { orderedActionsArraySelector } from 'Redux/Selectors/ActionsSelectors';
 
 export default function ActionButton() {
 	const dispatch = useDispatch();
-	const isActionsTabMounted = useSelector(state => state.MountedComponents.ActionsTab);
+	const isActionsTabMounted = useSelector(orderedActionsArraySelector);
```

Comment:
I believe you accidentally used the actions selector instead of the `mounteComponents` one...
(btw ActionsTab defaults to be mounted and there's no way to unmount it at the moment.)

Reply:
sorry about that.